LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2019 Предраг Николић
+Copyright (c) 2019-2020 Предраг Николић
```

Comment:
Feel free to put SublimeLSP instead of my name. GitHub probably put my name when I generated the License.

Reply:
I guess https://github.com/sublimelsp/LSP-html/pull/10/commits/47cd077aedc6df8dcebd82023012f1c5d7e29af1 would make both side happy :smiley: ?